### PR TITLE
add accessibility guidelines page to gatsby

### LINKED ISSUE #2206: Add Accessibility Docs to Gatsby - Accessibility Guidelines NDS
**User Story:**
As Spark we want to copy over Sparks Accessibility to Gatsby so that it is represented as it is today.

**Notes:**
https://sparkdesignsystem.com/docs/accessibility

**AC:**
- I will see accessibility doc in Gatsby
- Accessibility will live under Principles

### PR DESCRIPTION
**Pull request by guern1kn**
## What does this PR do?
Adds Accessibility Guidelines page to the Gatsby site.

### Associated Issue 
Fixes #2206 

### Documentation
 - [x] Update Spark Docs Gatsby

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [x] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [x] Apple Safari
  - [ ] Apple Safari (Mobile)

### CODE REVIEW

File: src/pages/principles/accessibility-guidelines.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+- ##### Text Alternatives
+
+    Provide alternatives for non-text content
+    (e.g., images) so that the content is
+    accessible for all users.
+
+- ##### Time-Based Media
+
+    Provide an alternative (e.g., transcript)
```
Comment (afebbraro) [inline src/pages/principles/accessibility-guidelines.mdx]:
```suggestion
    Provide an alternative (e.g. transcript)
```

File: src/pages/principles/accessibility-guidelines.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    Make all functionality available from a keyboard.
+
+- ##### Timing
+
+    Provide enough time for users to read and use content.
+
+- ##### Seizures
+
+    Do not include design elements that are known to
+    cause seizures (e.g., rapid flashing).
```
Comment (afebbraro) [inline src/pages/principles/accessibility-guidelines.mdx]:
```suggestion
    cause seizures (e.g. rapid flashing).
```
Are these e.g. supposed to have a comma after the last period? I didnt think so at first but now that i see it repeated throughout this content, I'm second guessing myself haha

Comment (guern1kn) [reply src/pages/principles/accessibility-guidelines.mdx]: No idea. This was directly copy and pasted from what is on our current site. I'll do a quick search

Comment (guern1kn) [reply src/pages/principles/accessibility-guidelines.mdx]: @afebbraro  AP Style says that it should have a comma.

Comment (afebbraro) [reply src/pages/principles/accessibility-guidelines.mdx]: Thanks so much for checking!!! Thats good to know.

**Pull request merged**